.eslintrc.json

```diff
@@ -54,6 +60,8 @@
         "react/jsx-filename-extension": "error",
         "react/prefer-stateless-function": "warn",
         "react/sort-comp": "warn",
-        "react/jsx-max-props-per-line": [1, { "when": "multiline" }]
+        "react/jsx-max-props-per-line": [1, { "when": "multiline" }],
+        "import/namespace": "off",
+        "import/default": "off"
```

Comment:
I am equally fine with Sanjo's approach, if we can fix the warnings.

In case we do need separate configs, I think I would prefer a few lines of messy gulp code over having to keep two folder trees in sync.